front-end/src/renderer/pages/RestoreKey/RestoreKey.vue

```diff
@@ -202,6 +202,12 @@ onBeforeUnmount(() => {
   user.recoveryPhrase = null;
 });
 
+onBeforeMount(() => {
```

Comment:
This is to skip the recovery phrase input step if the recovery phrase is still cached? Is that what we want? Or should we allow the user to see the cached phrase and use a different phrase if so desired?

Reply:
I see the logic was just moved from Step 1, good catch there. I'm just wondering if this is what we want the flow to be (auto skip the new Step 1 if phrase is cached)

Reply:
We won't skip the recovery step, we will only prefill the boxes

Reply:
done